bookkeeper-dist/src/main/resources/LICENSE-all.bin.txt

```diff
@@ -315,6 +315,8 @@ Apache Software License, Version 2.
 - lib/com.google.errorprone-error_prone_annotations-2.1.2.jar [48]
 - lib/org.apache.yetus-audience-annotations-0.5.0.jar [49]
 - lib/org.jctools-jctools-core-2.1.2.jar [50]
+- lib/org.apache.httpcomponents-httpclient-4.5.5.jar [51]
```

Comment:
It turns out in current master that we already have these jars on the server.

Maybe we can accept them now even for the client